tests/playground.html

```diff
@@ -347,6 +347,18 @@
   }
 }
 
+function changeRenderingConstant(value) {
+  var type = document.getElementById('rendering-constant-selector').value;
+  if (type == 'fontSize') {
```

Comment:
Might be nice to check if there is already a fontStyle on the theme so when someone changes the size the font family will not change back to the default.

Reply:
This is for playground testing, reloading the page will get you back to the defaults.